src/shared-components/typography/index.ts

```diff
@@ -93,15 +93,33 @@ export const style = {
   link: linkStyle,
 } as const;
 
-const Button = styled.span(buttonStyle);
-const Caption = styled.p(captionStyle);
-const Display = styled.h1(displayStyle);
-const ErrorComponent = styled.p(errorStyle);
-const Heading = styled.h2(headingStyle);
-const Label = styled.label(labelStyle);
-const Link = styled.a(linkStyle);
-const Success = styled.p(successStyle);
-const Title = styled.h3(titleStyle);
+const Button = styled.span`
+  ${({ theme }) => buttonStyle(theme)}
+`;
+const Caption = styled.p`
+  ${({ theme }) => captionStyle(theme)}
+`;
+const Display = styled.h1`
+  ${({ theme }) => displayStyle(theme)}
+`;
+const ErrorComponent = styled.p`
+  ${({ theme }) => errorStyle(theme)}
+`;
+const Heading = styled.h2`
+  ${({ theme }) => headingStyle(theme)}
+`;
+const Label = styled.label`
+  ${({ theme }) => labelStyle(theme)}
+`;
+const Link = styled.a`
+  ${linkStyle()}
```

Comment:
Left this without arguments for now, but wanted the function signatures to be identical.